unified-scheduler-logic/src/lib.rs

```diff
@@ -626,15 +626,21 @@ const_assert_eq!(mem::size_of::<UsageQueue>(), 8);
 pub struct SchedulingStateMachine {
     unblocked_task_queue: VecDeque<Task>,
     active_task_count: ShortCounter,
```

Comment:
maybe a comment on `active_task_comment` to clarify the meaning. afaict it is either buffered or executing?

Reply:
i added comment as part of this review comment addressing: https://github.com/anza-xyz/agave/pull/5169#discussion_r1991468687: 55da9903a9f53954459f74842c31cc7d482eed1c